master/getting-started/docker/installation/requirements.md

```diff
@@ -24,7 +24,9 @@ To use Calico as a Docker network plugin, the Docker daemon must be configured
 with a cluster store.  If using etcd as a cluster store,
 configure the `cluster-store` on the Docker daemon to `etcd://<ETCD_IP>:<ETCD_PORT>`,
 replacing `<ETCD IP>` and <ETCD_PORT> with the appropriate address and client
```

Comment:
While you're here, can you fix this inconsistency where ETCD IP is in code ticks but ETCD_PORT is not?  Also the missing underline in ETCD IP